schemas/dlhub.json

```diff
@@ -8,13 +8,22 @@
             "type": "string",
             "description": "Version of the DLHub service"
         },
+        "publication_date": {
+            "type": "integer",
+            "description": "Time when this model was submitted to DLHub, expressed as a UNIX timestamp (seconds from the Unix Epoch)"
+        }, 
         "domains": {
             "type": "array",
             "description": "Scientific application areas of the artifact",
             "item_type": {
                 "type": "string"
             }
         },
+        "owner": {
+            "type": "string",
+            "description": "Globus ID of the owner of the model",
```

Comment:
Is the `owner` going to need to be user-friendly? Globus IDs are useful, but unwieldy for end-users. I'm specifically wondering about searches. Will users want to search for/retrieve models owned by someone else, using the unique identifier for the model (`owner`+`name`)? If not (such as if user will only search by `name` and will accept receiving multiple models back in case of `name` collision), then this is fine.

Reply:
No worries, Ryan convinced me to make the owner human-friendly :)

And, yes, we are OK with having `name` be non-unique